ml-agents-envs/mlagents/envs/local_env_manager.py

```diff
@@ -0,0 +1,84 @@
+from typing import Any, Dict, List
+
+from mlagents.envs.base_unity_environment import BaseUnityEnvironment
+from mlagents.envs.env_manager import EnvManager, StepInfo
+from mlagents.envs.timers import timed
+from mlagents.envs import ActionInfo, BrainParameters
+
+
+class LocalEnvManager(EnvManager):
+    """
+    Simple implementation of the EnvManager interface that only handles one BaseUnityEnvironment at a time.
+    This is generally only useful for testing; see SubprocessEnvManager for a production-quality implementation.
+    """
+
+    def __init__(self, env: BaseUnityEnvironment):
+        super().__init__()
+        self.env = env
+        self.previous_step: StepInfo = StepInfo(None, {}, None)
+        self.previous_all_action_info: Dict[str, ActionInfo] = {}
+
+    def get_last_steps(self):
```

Comment:
This function should be unnecessary against the latest `develop`

Reply:
Removed.